File: benchmark/comparison/results/blind_ts/08.ts
function factorial(n: number): string {
  if (n < 0) return "Err(negative input)";
  let result = 1;
  for (let i = 2; i <= n; i++) result *= i;
  return `Ok(${result})`;
}

console.log(factorial(0));
console.log(factorial(5));
console.log(factorial(-1));

File: benchmark/comparison/results/blind_py/08.py
def factorial(n):
    if n < 0:
        return "Err(negative input)"
    result = 1
    for i in range(1, n + 1):
        result *= i
    return f"Ok({result})"

print(factorial(0))
print(factorial(5))
print(factorial(-1))
